src/main/scala/nl.knaw.dans.easy.dd2d/dataverse/HttpSupport.scala

```diff
@@ -47,12 +47,14 @@ trait HttpSupport extends DebugEnhancedLogging {
 
   protected def httpPostMulti(uri: URI, file: File, optJsonMetadata: Option[String] = None, headers: Map[String, String] = Map()): Try[HttpResponse[Array[Byte]]] = Try {
     trace(())
-    val parts = MultiPart(name = "file", filename = file.name, mime = "application/octet-stream", data = file.byteArray) +:
-      optJsonMetadata.map {
-        json => List(MultiPart(data = json.getBytes(StandardCharsets.UTF_8), name = "jsonData", filename = "jsonData", mime = "application/json"))
-      }.getOrElse(Nil)
-
-    Http(uri.toASCIIString).postMulti(parts: _*)
+    optJsonMetadata.map {
```

Comment:
Why do you now call postMulti twice?

Reply:
It was because after using the input stream I got an erroneous result in Dataverse dataset. I didn't remember anymore what the problem was, and therefore tested it again with the original one postMulti call.  And behold, there was no problem anymore.  There must then have been something else in the test configuration that caused the error.  I have now changed the code back to one postMulti call.